server/cpho/fixtures/periods.yaml

```diff
@@ -13,6 +13,7 @@
   fields:
     year: 2023
     year_type: calendar
+    is_current: True
```

Comment:
the fixtures were updated for dev purposes but I think I'll use the django-admin in prod